hack/update_eth_clients.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [ "$#" -lt 2 ]; then
+  echo "Usage: $0 <network> <commit_sha> [workspace_file]" >&2
+  echo "  network: mainnet | holesky | sepolia | hoodi" >&2
```

Comment:
This is just a script to update the WORKSPACE in the future for ethclient package verifying configs